lab-michael/server.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const express = require('express');
+const morgan = require('morgan');
+const jsonParser = require('body-parser').json();
+
+const createError = require('http-errors');
+
+const RecipeApp = express();
+const router = express.Router();
+
+RecipeApp.use(morgan('dev'));
+RecipeApp.use(jsonParser);
+
+RecipeApp.use((err, req, res, next) => {
```

Comment:
You also want to move this error handling to a separate module, and then import that module to then be passed to app.use.